lib/serve/rest-api/src/api/routes.py

```diff
@@ -30,6 +31,9 @@
 router.include_router(models.router, prefix="/v1", tags=["models"], dependencies=[Depends(security)])
```

Comment:
Are we going to be marking the v1 endpoints as depreciated?

Reply:
There's not going to be a lot of time between them being deprecated and us releasing the next revision, so I think for now, we can just leave as-is